cmd/build.go

```diff
@@ -52,16 +52,25 @@ func newBuildCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	// buildCmd provides the ability run local builds, or setup/delete Tekton builds, for an appsody project
 	var buildCmd = &cobra.Command{
 		Use:   "build",
-		Short: "Locally build a docker image of your appsody project",
-		Long:  `This allows you to build a local Docker image from your Appsody project. Extract is run before the docker build.`,
+		Short: "Locally build a container image of your Appsody project",
```

Comment:
I think the first part of the long description would be better here, ie "Build a local container image of your Appsody project." 
"Locally build" just sounds a bit odd :)